### ISSUE TITLE
error while loading shared libraries from /usr/local/lib, dir not in path under Linux Mint

### ISSUE THREAD

**Issue by der-den** [nih-at/libzip#427]
Hi,

I tried to install libzip from sources..

"make install" drop the libs to /usr/local/lib/ and this path is not in the lib-path of Linux Mint 21.3.

After some minutes of search, i found, i must use ldconfig to fix this and get libzip-tools running.

Maybe you can check and add this to your install script !? Or do i something wrong?

Regards

**Comment by 0-wiz-0** [comment]
On Linux, it is not usual to add the runtime path (rpath) into binaries (I don't know why; on other operating systems it is common). For this reason, we explicitly disable this CMake feature on Linux, see [here](https://github.com/nih-at/libzip/blob/f0eff80889f1b0791e91516d2d84e8d5b84a1e02/CMakeLists.txt#L289).
Users installing programs on Linux are usually aware of this and either change the ldconfig file or use a different installation prefix that's already covered by the existing ldconfig; or they use their distro's package manager (in your case I think that would be `sudo apt install libzip-dev`).
The libzip installation definitely shouldn't touch the ldconfig config file.
In summary, I don't think there's anything libzip should do differently here.

You can
1. change the ldconfig file as you did
2. install into `/usr` (or some other path that's covered by your ldconfig config file)
3. try enabling the RPATH handling in `CMakeLists.txt`
4. or use the libzip-dev Mint package.
